src/ride/track.c

```diff
@@ -4880,15 +4880,162 @@ void game_command_remove_track(int *eax, int *ebx, int *ecx, int *edx, int *esi,
 	return;
 }
 
+uint32 loc_6CDE51() {
+	// mov     ebx, 80000000h
+	return 0x80000000;
+}
+
+uint32 loc_6CDE4C() {
+	// pop     edi
+	// pop     edx
+	// pop     ecx
+	// pop     ebx
+	// pop     eax
+	return loc_6CDE51();
+}
+
+uint32 loc_6CDE43() {
+	// mov     game_command_error_text, 0
+	RCT2_GLOBAL(RCT2_ADDRESS_GAME_COMMAND_ERROR_TEXT, rct_string_id) = 0;
+	
+	return loc_6CDE51();
+}
+
+uint32 loc_6CDE38() {
+	// mov     game_command_error_text, 0
+	RCT2_GLOBAL(RCT2_ADDRESS_GAME_COMMAND_ERROR_TEXT, rct_string_id) = 954; // empty string?
+	
+	return loc_6CDE51();
+}
+
+uint32 loc_6CDE2D() {
+	// mov     game_command_error_text, 0
+	RCT2_GLOBAL(RCT2_ADDRESS_GAME_COMMAND_ERROR_TEXT, rct_string_id) = STR_CONSTRUCTION_NOT_POSSIBLE_WHILE_GAME_IS_PAUSED;
+	
+	return loc_6CDE51();
+}
+
+uint32 loc_6CDE22() {
+	// mov     game_command_error_text, 0
+	RCT2_GLOBAL(RCT2_ADDRESS_GAME_COMMAND_ERROR_TEXT, rct_string_id) = STR_RIDE_CANT_BUILD_THIS_UNDERWATER;
+	
+	return loc_6CDE4C();
+}
+
+uint32 loc_6CDE17() {
+	// mov     game_command_error_text, 0
+	RCT2_GLOBAL(RCT2_ADDRESS_GAME_COMMAND_ERROR_TEXT, rct_string_id) = STR_CAN_ONLY_BUILD_THIS_ABOVE_GROUND;
+	
+	return loc_6CDE4C();
+}
+
+uint32 loc_6CD920(int *eax, int *ebx, int *ecx, int *edx, int *esi, int *edi, int *ebp) {
+
+	// test    di, 0Fh
+	// jnz     loc_6CDE38
+	if (((*edi & 0xFFFF) & 0xF) != 0) {
+		loc_6CDE38();
+	}
+	
+	// test    bl, 1
+	// jz      short loc_6CD954
+	if (((*ebx & 0xFF) & 0x1) == 0) {
+		loc_6CD954();
+	}
+	
+	// test    bl, 8
+	// jnz     short loc_6CD954
+	if (((*ebx & 0xFF) & 0x8) != 0) {
+		loc_6CD954();
+	}
+	
+	// push    edx
+	
+	// mov     dx, di
+	//dx = di; // EDX = EDX with DX set to zero, then add DI
+	uint32 temp_edx = (*edx & 0xFFFF0000) + (*edi & 0xFFFF);
+	
+	// call    footpath_remove_litter
+	footpath_remove_litter(*eax, *ecx, temp_edx);
+	
+	// push    eax
+	// push    ebx
+	// push    ecx
+	
+	// and     ax, 0FFE0h
+	uint32 temp_x = (*eax & 0xFFFF0000) + (*eax & 0x0FFE0);
```

Comment:
By anding with 0xFFE0 you are looking at a specific quadrant rather than an x, y coordinate. Generally if you see & 0xFFE0 you know that you are dealing with coordinates of a map element.